lua/gitlab/init.lua

```diff
@@ -22,10 +22,11 @@ return {
     end
     server.build() -- Builds the Go binary if it doesn't exist
     state.setPluginConfiguration() -- Sets configuration from `.gitlab.nvim` file
-    state.merge_settings(args) -- Sets keymaps and other settings from setup function
+
+    if not state.merge_settings(args) then -- Sets keymaps and other settings from setup function
```

Comment:
If the merge_settings function returns false it means there was a fatal error and we do not try to keep running the setup function

Reply:
shouldn't there be a notification for user ?

Reply:
my bad I just noticed that message is in function.